haybaler_taxonomy.py

```diff
@@ -88,10 +125,14 @@ def main(input_file, input_path):
         csv.insert(loc=0, column='genus', value=genus_series.values)
         save_csv(csv, input_path, input_file)
     else:
+        nan_list = set(taxonomy[taxonomy["TaxID"].isna()]["Name"].to_list())  # list of names that produce NAN
+        genus_series = pd.Series(genus)
+        csv.insert(loc=0, column='genus', value=genus_series.values)
+        print(csv[csv['genus'].isin(nan_list)]["genus"])  # print everything that didn't work with pytaxonkit
+        # print(taxonomy[taxonomy["TaxID"].isna()])  # print everything that didn't worked with pytaxonkit (old)
         total_chr = len(genus)
         chr_not_work = len(taxonomy[taxonomy["TaxID"].isna()])
         chr_work = total_chr - chr_not_work
```

Comment:
Again, I'd count both, not infer (to avoid error issues being caught and numbers not matching).